stable/ambassador/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 0.60.3
+appVersion: 0.61.0
 description: A Helm chart for Datawire Ambassador
 name: ambassador
-version: 2.4.0
```

Comment:
I think this can be 2.4.1 rather than 2.5.0 -- we're not changing the structure of the Helm chart or anything.

Reply:
Fixed